migrations/versions/0318_update_postage_constraint.py

```diff
@@ -0,0 +1,76 @@
+"""
+
+Revision ID: 0318_update_postage_constraint
+Revises: 0317_uploads_for_all
+Create Date: 2020-03-11 12:01:41.533192
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0318_update_postage_constraint'
+down_revision = '0317_uploads_for_all'
+
+
+def upgrade():
+    op.drop_constraint('chk_notifications_postage_null', 'notifications')
+    op.drop_constraint('chk_notification_history_postage_null', 'notification_history')
+    op.drop_constraint('chk_templates_postage', 'templates')
+    op.drop_constraint('chk_templates_history_postage', 'templates_history')
+
+    op.execute("""
+        ALTER TABLE notifications ADD CONSTRAINT "chk_notifications_postage_null"
+        CHECK (
+            CASE WHEN notification_type = 'letter' THEN
+                postage is not null and postage in ('first', 'second', 'international')
+            ELSE
+                postage is null
+            END
+        )
+        NOT VALID
+    """)
+    op.execute("""
```

Comment:
I don't think we need to check constraint on `NOTIFICATION_HISTORY` because the data is copied from `NOTIFICATIONS` as long as that does have the constraint we should be alright.

Reply:
`notification_history` has already got a check constraint on the postage - I could just remove it instead of updating it. If we're going to remove it, then maybe we should also remove the constraint from `templates_history` to be consistent.

Reply:
I've dropped the constraint from `notification_history` since the table is so large, but kept the constraint on the other 3 tables like we discussed